nnpdf_data/nnpdf_data/commondata/ATLAS_WPWM_7TEV_46FB/metadata.yaml

```diff
@@ -18,12 +18,12 @@ implemented_observables:
     description: Drell-Yan Rapidity Distribution
     label: ATLAS $W$ 7 TeV 2011 Central selection
     units: ''
-  process_type: EWK_RAP
+  process_type: EWK_RAP #I can't see this as a process in process_options.py
```

Comment:
I can't see an option for EWK_RAP in [https://github.com/NNPDF/nnpdf/blob/master/validphys2/src/validphys/process_options.py](https://github.com/NNPDF/nnpdf/blob/master/validphys2/src/validphys/process_options.py)
but it is an option here: [https://docs.nnpdf.science/data/exp-data-files.html](https://docs.nnpdf.science/data/exp-data-files.html). I just wanted to check that this means it's okay to use as a process.

Reply:
The documentation is outdated and hopefully someone rewrites it ^^U

But indeed, you need to use one of the ones in https://github.com/NNPDF/nnpdf/blob/master/validphys2/src/validphys/process_options.py, so for this it would be DY_W_ETA I guess.

Make sure to change the `kinematics_override` to be the identity (this is an old key that we want to remove, but we need to wait until all of the files are ported).